src/main/java/com/juniorstart/juniorstart/controller/UserController.java

```diff
@@ -24,17 +26,23 @@
 @RestController
 public class UserController {
     final private UserService userService;
-
+    final private UserDao userDao;
     @Autowired
-    public UserController(UserService userService) {
+    public UserController(UserService userService,UserDao userDao) {
         this.userService = userService;
+        this.userDao=userDao;
     }
 
     @GetMapping("/user/me")
     @PreAuthorize("hasRole('USER')")
     public User getCurrentUser(@CurrentUser UserPrincipal userPrincipal) {
         return userService.getCurrentUser(userPrincipal);
     }
+    
+    @GetMapping("/all")  
+    public List<User> all() {
+        return userDao.findAll();
```

Comment:
To już jest za dużo. Niżej jest funkcja getUsers która robi to samo. Gdy nie będzie podany żaden argument to zwróci wszystkich użytkowników :)